test/hl7_message_test.exs

```diff
@@ -72,7 +72,7 @@ defmodule HL7MessageTest do
     new_msg = raw_text_with_garbage |> HL7.Message.new()
     roundtrip = new_msg |> to_string()
     assert roundtrip == raw_text
-    assert new_msg.fragments == ["garbage text"]
+    assert new_msg.fragments == [["garbage text"]]
```

Comment:
Curious why this changed.

Reply:
It was how I was chucking unparseable HL7 junk off to the side (which ended up being slightly different in the new version). I might try to figure out a better way to handle it -- but I'm not really sure this test even makes sense as is (since neither version keeps the garbage in the right place I noticed if you want a roundtrip, just appends it at the end). I would need to make structural changes to the library to support more things -- and I wanted to minimize my changes here for safety.